storage/replica.go

```diff
@@ -193,11 +196,12 @@ var _ client.Sender = &Replica{}
 // NewReplica initializes the replica using the given metadata.
 func NewReplica(desc *roachpb.RangeDescriptor, rm *Store) (*Replica, error) {
 	r := &Replica{
-		store:       rm,
-		cmdQ:        NewCommandQueue(),
-		tsCache:     NewTimestampCache(rm.Clock()),
-		sequence:    NewSequenceCache(desc.RangeID),
-		pendingCmds: map[cmdIDKey]*pendingCmd{},
+		store:          rm,
+		cmdQ:           NewCommandQueue(),
+		tsCache:        NewTimestampCache(rm.Clock()),
+		sequence:       NewSequenceCache(desc.RangeID),
+		pendingCmds:    map[cmdIDKey]*pendingCmd{},
+		raftEntryCache: newRaftEntryCache(defaultRaftEntryCacheSize),
```

Comment:
Replicas down maybe unusual, and the eviction can prevent mem usage grow infinitly, but the Mem consumption also depands on the size of key/value,  this maybe more difficult to control, it seems difficult to make a decision, do you have any suggestion.